dkube/secrets.env

```diff
@@ -1,5 +1,4 @@
 KATIB_MYSQL_DB_DATABASE=katib
 KATIB_MYSQL_DB_HOST=katib-mysql
 KATIB_MYSQL_DB_PORT=3306
-DB_USER=root
-DB_PASSWORD=test
\ No newline at end of file
+DBCONFIG_USER=root
```

Comment:
this can be removed too correct.

Reply:
I tried using hard coded value for username in the config map patches. But there is a bug in patchesStrategicMerge which won't allow you to over write a valueFrom with value in envs. We can only patch with valueFrom.